scripts/completion-tests/test-completion.sh

```diff
@@ -34,10 +34,26 @@ export -f set_shell_debug_level
 set_shell_debug_level 2
 
 BINARY_NAME=helm
-BINARY_ROOT=${ROBOT_HELM_PATH:-${SCRIPT_DIR}/../../../helm/bin}
-BINARY_PATH_DOCKER=${BINARY_ROOT}/../_dist/linux-amd64
+HELM_DIR=${SCRIPT_DIR}/../../.acceptance
+HELM_REPO=${HELM_DIR}/helm
+BINARY_ROOT=${HELM_REPO}/bin
+BINARY_PATH_DOCKER=${HELM_REPO}/_dist/linux-amd64
```

Comment:
why `BINARY_PATH_DOCKER` ? why `DOCKER` ?

Reply:
They test diff shell in diff docker containers, and docker is an upstream choice, I just changed right-hand-side value.